_posts/2017-06-15-grpc-load-balancing.md

```diff
@@ -0,0 +1,190 @@
+## Introduction
+
+This document describes various load balancing scenarios seen when deploying gRPC. If you use [gRPC](http://grpc.io) with multiple backends, this document is for you.
+
+A large scale gRPC deployment typically has a number of identical back-end instances, and a number of clients. Each server has a certain capacity. Load balancing is used for distributing the load from clients optimally across available servers.
+
+### Why gRPC?
```

Comment:
We need to add or remove something here otherwise it's a bit "Why this section?" - there's nothing here that we've explicitly linked to load balancing (though we mention HTTP/2 advantages later on in the application-level LB section) .